content/en/guides/configuration-glossary/configuration-loading-indicator.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: 'The loading indicator Property'
-description: Show fancy loading indicator while SPA page is loading!
+description: Show fancy loading indicator while page is loading!
 menu: loadingIndicator
 category: configuration-glossary
 position: 16
 ---
 
-> Show fancy loading indicator while SPA page is loading!
+> Show fancy loading indicator while page is loading!
 
-When running Nuxt.js in SPA mode, there is no content from the server side on the first page load. So, instead of showing a blank page while the page loads, we may show a spinner.
+When running Nuxt.js with `ssr` set to `false`, there is no content from the server side on the first page load. So, instead of showing a blank page while the page loads, we may show a spinner.
```

Comment:
```suggestion
Without Server Side Rendering (when `ssr` option is `false`), there is no content from the server side on the first page load. So, instead of showing a blank page while the page loads, we may show a spinner.
```